lib/util/constants.ts

```diff
@@ -10,22 +10,17 @@
  */
 export type LongRunningOperationStates = "InProgress" | "Succeeded" | "Failed" | "Canceled";
 
-const Constants = {
-
-  /**
-   * The default language in the request header.
-   *
-   * @const
-   * @type {string}
-   */
-  DEFAULT_LANGUAGE: "en-us",
-
-  /**
-   * The ms-rest-azure version.
-   * @const
-   * @type {string}
-   */
-  msRestAzureVersion: "0.1.0"
-};
+/**
+ * The default language in the request header.
+ *
+ * @const
+ * @type {string}
+ */
+export const DEFAULT_LANGUAGE = "en-us";
 
-export default Constants;
+/**
+ * The ms-rest-azure version.
+ * @const
+ * @type {string}
+ */
+export const msRestAzureVersion = "0.1.0";
```

Comment:
Down the road maybe we can think of a way to keep this number accurate (which is supposed to be the same as whatever we publish).. I think we plan on using this for ARM telemetry so that will be valuable.

Reply:
I agree that we should figure out how to automatically update this. We might only be able to keep the major and minor versions synchronized (since the patch version is generated by our publish job).